tests/integration/udf/test_least_frequent.py

```diff
@@ -0,0 +1,32 @@
+"""
+Tests for least frequent UDF
+"""
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize("source_type", ["snowflake", "spark"], indirect=True)
+@pytest.mark.parametrize(
+    "counts, expected",
+    [
+        ({}, None),
+        ({"__MISSING__": None}, None),
```

Comment:
ooc: where does `__MISSING__` come from?

Reply:
Missing values in the category column are converted to that special value before cross aggregation: https://github.com/featurebyte/featurebyte/blob/88290a599c164569ebc03c045ee045efa0eb1d7b/featurebyte/query_graph/sql/adapter/base.py#L264-L273

Reply:
gotcha, can consider using that string constant here then if you want!